server/db/sqlite/storage.go

```diff
@@ -417,6 +417,11 @@ func (me *DB) CreateDB() error {
 	})
 }
 
+func (me *DB) Close() error {
+	log.Println("Closing db")
```

Comment:
do we need to log here?

Reply:
I found it helpful to see if it _doesn't_ get called.